config.go

```diff
@@ -231,6 +238,7 @@ func DefaultWANConfig() *Config {
 	conf.ProbeInterval = 5 * time.Second
 	conf.GossipNodes = 4 // Gossip less frequently, but to an additional node
 	conf.GossipInterval = 500 * time.Millisecond
+	conf.HandoffQueueDepth = 1024
```

Comment:
You can remove this and the one on line 258 since these both inherit from the LAN config, that way these only specify things that change for WAN and local.